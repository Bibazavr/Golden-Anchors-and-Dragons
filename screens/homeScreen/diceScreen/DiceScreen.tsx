import * as React from "react";
import {StyleSheet, View} from "react-native";

import {Dice10} from "./components/dices";


export const DiceScreen = () => {
    return (
        <View style={stylesDiceScreen.container}>
            <Dice10/>
        </View>
    )
}

const stylesDiceScreen = StyleSheet.create({
    container: {
        flex: 1,
        flexDirection: "row",
        alignItems: "center",
        justifyContent: "center",
        backgroundColor: '#fff',
    },
});
