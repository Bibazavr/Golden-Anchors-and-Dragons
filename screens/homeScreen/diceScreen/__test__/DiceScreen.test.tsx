import * as React from "react";
import renderer from "react-test-renderer";
import {DiceScreen} from "../DiceScreen";

it(`not toBeUndefined`, () => {
    const tree = renderer.create(<DiceScreen/>).toJSON();

    expect(tree).not.toBeUndefined();
});