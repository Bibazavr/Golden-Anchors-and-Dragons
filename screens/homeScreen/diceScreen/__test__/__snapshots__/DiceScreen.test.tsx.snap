// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  >
    <View
      style={
        Object {
          "borderRadius": 3,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#fff",
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#2089dc",
              "borderColor": "#2089dc",
              "borderRadius": 3,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontSize": 18,
                "paddingBottom": 1,
                "paddingTop": 2,
                "textAlign": "center",
              }
            }
          >
            4
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
